PWGHF/HFC/Tasks/taskCorrelationDMesonPairs.cxx

```diff
@@ -47,9 +47,29 @@ namespace
 {
 enum CandidateTypeSel {
   SelectedD = 0, // This particle is selected as a D
-  SelectedDbar,  // This particle is selected as a Dbar
+  SelectedDBar,  // This particle is selected as a Dbar
   TrueD,         // This particle is a true D
-  TrueDbar       // This particle is a true Dbar
+  TrueDBar       // This particle is a true Dbar
+};
+
+enum DMesonType {
+  DefaultD = 0, // Default value
+  SignalD,      // This particle is a signal D
+  ReflectedD,   // This particle is a reflected D
+  BkgD          // This particle is background of D
+};
+
+enum DBarMesonType {
+  DefaultDBar = 0, // Default value
+  SignalDBar,      // This particle is a signal Dbar
+  ReflectedDBar,   // This particle is a reflected Dbar
+  BkgDBar          // This particle is background of Dbar
+};
```

Comment:
Do you need to separate D and Dbar?

Reply:
I needed it before, but not anymore, true